Content.Shared/Humanoid/NamingSystem.cs

```diff
@@ -40,6 +40,9 @@ public string GetName(string species, Gender? gender = null)
                 case SpeciesNaming.FirstDashFirst:
                     return Loc.GetString("namepreset-firstdashfirst",
                         ("first1", GetFirstName(speciesProto, gender)), ("first2", GetFirstName(speciesProto, gender)));
+                case SpeciesNaming.LastFirst: // DeltaV: Rodentia name scheme
+                    return Loc.GetString("namepreset-lastfirst",
+                        ("last", GetLastName(speciesProto)), ("first", GetFirstName(speciesProto, gender)));
```

Comment:
Shouldn't you just swap where you put the name lists?

Reply:
first names are gendered, last names are not